src/app/shared/services/HttpService/HttpService.ts

```diff
@@ -0,0 +1,29 @@
+import VueRouter from 'vue-router';
+import axios, { AxiosError, AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios';
+import { Store } from 'vuex';
+import { IState } from '../../../state';
+import { setupResponseInterceptor } from './setupResponseInterceptor';
+import { setupRequestInterceptor } from './setupRequestInterceptor';
+
+export interface IHttpService extends AxiosInstance {
+  store?: Store<IState>;
+  router?: VueRouter;
+}
+
+let baseURL;
+
+/* istanbul ignore if */
+if (!CLIENT) {
+  const config = require('../../../config/AppConfig');
+  baseURL = config.api.baseUrl;
+}
+
+export const HttpService: IHttpService = axios.create({
+  baseURL,
```

Comment:
I don't know if this will have a negative impact on calling external api's. do we have this currently in the project?

Reply:
`baseURL` will be prepended to `url` unless `url` is absolute. it should be fine, what do you think?

Reply:
didn't know that it works like that. Then we can get rid of the `isExternalUrl` logic completely if axios handle it for us.